.github/workflows/ci.yml

```diff
@@ -88,8 +88,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-        with:
-          submodules: "recursive"
+      - name: Clone coreJSON 
+        run: git clone https://github.com/FreeRTOS/coreJSON.git 
```

Comment:
You can revert this back now that you have a `.gitmodules` file. The recursive checkout will clone CoreJSON for you